### PR TITLE
feat(flags): implement get_feature_flag endpoint in SDK compliance adapter

### PR DESCRIPTION
**Pull request by andehen**
## Problem 
The SDK compliance adapter doesn't expose a way to evaluate feature flags, which is needed for cross-SDK compliance testing of flag evaluation.

## Changes
- Added a `/get_feature_flag` POST endpoint to the compliance adapter that wraps `client.get_feature_flag()`
- Supports `person_properties`, `groups`, `group_properties`, `disable_geoip`, and `force_remote` parameters
- Maps `force_remote` to the inverse of `only_evaluate_locally` for a cleaner external API

## Testing
- Call the `/get_feature_flag` endpoint with a `key` and `distinct_id` after initializing the SDK via the adapter
- Verify flag evaluation with various combinations of person/group properties
- Verify error responses when `key` or `distinct_id` are missing or SDK is not initialized

### CODE REVIEW

File: sdk_compliance_adapter/adapter.py [hunk 1]
```diff
@@ -360,6 +360,46 @@ def get_state():
...
+
+        value = state.client.get_feature_flag(
+            key,
+            distinct_id,
+            person_properties=person_properties,
+            groups=groups,
+            group_properties=group_properties,
+            disable_geoip=disable_geoip,
+            only_evaluate_locally=not force_remote,
+        )
```
[the remark this answers is not in the document]
Comment (marandaneto) [reply sdk_compliance_adapter/adapter.py]: @andehen something to be fixed?

**Review by marandaneto — APPROVED** [review]

**Comment by andehen** [comment]
@marandaneto Thanks! Wasn't expecting a review yet. Created this as part of testing the SDK harness addition upstream so I could verify that it worked as intended. But I think it's ready to go as it is now, so thanks! 👍 

FYI the plan forward is to add such an adapter to all SDK's that support flags (all I guess?), so we can automatically test that the comply with the flags API and behave similarly wrt. to edge cases, etc.

**Comment by marandaneto** [comment]
> @marandaneto Thanks! Wasn't expecting a review yet. Created this as part of testing the SDK harness addition upstream so I could verify that it worked as intended. But I think it's ready to go as it is now, so thanks! 👍
> 
> FYI the plan forward is to add such an adapter to all SDK's that support flags (all I guess?), so we can automatically test that the comply with the flags API and behave similarly wrt. to edge cases, etc.

we can merge it and fix if needed 
you can mark a pr as draft, prs got auto assigned because i removed the master branch in favor of main, and then i just reviewed :D 
thanks!
